bindings/python/crocoddyl/multibody/contacts/contact-1d.cpp

```diff
@@ -24,21 +30,23 @@ void exposeContact1D() {
       "The calc and calcDiff functions compute the contact Jacobian and drift (holonomic constraint) or\n"
       "the derivatives of the holonomic constraint, respectively.",
       bp::init<boost::shared_ptr<StateMultibody>, pinocchio::FrameIndex, double, std::size_t,
-               bp::optional<Eigen::Vector2d> >(
-          bp::args("self", "state", "id", "xref", "nu", "gains"),
+               bp::optional<Eigen::Vector2d, std::size_t, pinocchio::ReferenceFrame> >(
+          bp::args("self", "state", "id", "xref", "nu", "gains", "type", "pinRefFrame"),
```

Comment:
Set default argument here.

Reply:
Also rename the arguments as follows:
 - `type` to `mask`
 - `pinRefFrame` to `type`

This latter follows the standard used in other classes.

Reply:
@jcarpent  concerning the default argument I had

`( bp::args("self", "state", "id", "xref", "nu", "gains", "type"), bp::arg("pinRefFrame")=pinocchio::LOCAL ),`

but it seemed to make python unittest fail and caused an import error in python, I couldn't figure out why . I need to investigate this . Any clue?

Reply:
@cmastalli  addressed in [fd58261](https://github.com/loco-3d/crocoddyl/pull/1054/commits/fd582614a34b626d8cb0751df8132d31f643845b)